src/slurm_plugin/computemgtd.py

```diff
@@ -53,14 +61,18 @@ def __repr__(self):
         attrs = ", ".join(["{key}={value}".format(key=key, value=repr(value)) for key, value in self.__dict__.items()])
         return "{class_name}({attrs})".format(class_name=self.__class__.__name__, attrs=attrs)
 
-    @log_exception(log, "reading computemgtd config", catch_exception=IOError, raise_on_error=True)
+    @log_exception(log, "reading computemgtd config", catch_exception=Exception, raise_on_error=True)
     def _get_config(self, config_file_path):
         """Get computemgtd configuration."""
         log.info("Reading %s", config_file_path)
         config = ConfigParser()
         try:
-            config.read_file(open(config_file_path, "r"))
-        except IOError:
+            # Use subprocess based method with timeout to read shared file to prevent hanging when NFS is down
+            config_content = check_command_output(
```

Comment:
here again I'm concerned that when running this command through the shell we get some unexpected output that breaks our logic (e.g. is the shell profile sourced?can this add unexpected output?)